x-pack/test/apm_api_integration/tests/alerts/error_count_threshold.spec.ts

```diff
@@ -170,6 +170,10 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         expect(resp.hits.hits[0]._source).property('service.name', 'opbeans-java');
         expect(resp.hits.hits[0]._source).property('service.environment', 'production');
         expect(resp.hits.hits[0]._source).property('transaction.name', 'tx-java');
+        expect(resp.hits.hits[0]._source).property(
+          'error.grouping_key',
+          '2cb494454edc437679309dc190d692368ae7da490f45aff291fb6f7989437d4a'
```

Comment:
You can avoid hardcoding the error grouping key like:

```
const errorMessage = '[ResponseError] index_not_found_exception';
const errorGroupingKey = getErrorGroupingKey(errorMessage);

```suggestion
          errorGroupingKey
```

Reply:
thanks! updated.